docs/DeveloperGuide.md

```diff
@@ -230,6 +230,44 @@ The following activity diagram summarises what happens when a user executes the
   * Pros: Allows more flexibility, now the user can find persons whose expiry dates is not only 30 days.
   * Cons: Need to determine the range of days allowed for the user to enter, security concerns such as integer overflow could occur if user decides to perform malicious activities.
 
+### `sort`
+The `remind` command allows the user to view the profiles arranged in order of earliest to latest policy expiration date, with those profiles that have no policy data placed at the end of the late
```

Comment:
I think you forget to change this to sort command, will go ahead and change for you.

```suggestion
The `sort` command allows the user to view the profiles arranged in order of earliest to latest policy expiration date, with those profiles that have no policy data placed at the end of the late
```